src/scss/utilities/index.scss

```
@@ -0,0 +1,10 @@
+@charset "UTF-8";
+
+// Import full scss version of decanter to use variables etc
+@import 'decanter/core/src/scss/decanter';
```

```suggestion
@import 'decanter/core/src/scss/decanter-no-markup';
```

I think we want to stay away from rendering the whole thing when including the utilities.